### PR TITLE
user model

### PR DESCRIPTION
**Pull request by brac193**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  root to: redirect('/users/sign_up')
```
Comment (wbucko) [inline config/routes.rb]: Imo no need for that. Root is set in different PR to welcome page.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  validates :username, presence: true
```
Comment (wbucko) [inline app/models/user.rb:7]: Consider adding validation for email as well.

File: config/routes.rb (hunk 1 above)
Comment (brac193) [reply config/routes.rb]: removed

File: app/models/user.rb (hunk 2 above)
Comment (brac193) [reply app/models/user.rb:7]: according to https://github.com/plataformatec/devise/blob/master/lib/devise/models/validatable.rb#L31
isn't it enough?

File: config/routes.rb (hunk 1 above)
Comment (wbucko) [reply config/routes.rb]: Cool, git catch!

File: app/models/user.rb (hunk 2 above)
Comment (wbucko) [reply app/models/user.rb:7]: Cool, git catch!

**Review by wbucko — APPROVED** [review]

**Pull request merged**